### PR TITLE
Update eventcontent structure in RecoHI packages

### PR DESCRIPTION
**Pull request by jeongeun**
#### PR description:  
Update event content definitions to explicitly have RECO to be a superset of AOD and for FEVT to be a superset of RECO. 5 files changed : 
+ `RecoHI/Configuration/python/RecoHI_EventContent_cff.py`
+ `RecoHI/HiCentralityAlgos/python/RecoHiCentrality_EventContent_cff.py `
+ `RecoHI/HiEvtPlaneAlgos/python/RecoHiEvtPlane_EventContent_cff.py` 
+ `RecoHI/HiMuonAlgos/python/RecoHiMuon_EventContent_cff.py `
+ `RecoHI/HiTracking/python/RecoHiTracker_EventContent_cff.py `

FYI, `RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py`, `RecoHI/HiJetAlgos/python/RecoHiJets_EventContent_cff.py` files are work in progress and they will be updated in new PRs later (maybe separately).

(The previous tasks are [PR#29025](https://github.com/cms-sw/cmssw/pull/29025), [PR#29158](https://github.com/cms-sw/cmssw/pull/29158), [PR#29225](https://github.com/cms-sw/cmssw/pull/29225), [PR#29299](https://github.com/cms-sw/cmssw/pull/29299), [PR#29470](https://github.com/cms-sw/cmssw/pull/29470), [PR#29517](https://github.com/cms-sw/cmssw/pull/29517),  [PR#29535](https://github.com/cms-sw/cmssw/pull/29535), and  [PR#29571](https://github.com/cms-sw/cmssw/pull/29571) )

#### PR validation:
Event Content comparison check was also done and there is no change with these updates.
Tested in CMSSW_11_1_X, the basic test all passed in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html).

### CODE REVIEW

**Comment by perrotta** [comment]
please test

File: RecoHI/HiMuonAlgos/python/RecoHiMuon_EventContent_cff.py [hunk 1]
```diff
@@ -1,54 +1,52 @@
...
-    outputCommands = cms.untracked.vstring()
-    )
-
 #AOD content
 RecoHiMuonAOD = cms.PSet(
     outputCommands = cms.untracked.vstring()
     )
-
 #Add Isolation
 from RecoMuon.MuonIsolationProducers.muIsolation_EventContent_cff import *
```
Comment (perrotta) [inline RecoHI/HiMuonAlgos/python/RecoHiMuon_EventContent_cff.py:8]: Where are these ```RecoMuonIsolationXXX``` event contents used here, or in the calling configs?
If I am not wrong this ```import``` could even be removed here, if we want to clean up the config.
@mandrenguyen ?

Comment (mandrenguyen) [reply RecoHI/HiMuonAlgos/python/RecoHiMuon_EventContent_cff.py:8]: Indeed, it looks like nothing from that import is actually used.    If it's not doing anything, might as well remove it.

Comment (jeongeun) [reply RecoHI/HiMuonAlgos/python/RecoHiMuon_EventContent_cff.py:8]: Okay, I will remove it.

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- Structure of the RecoHI event content packages updated according to the plan, with no effect expected on the actual products saved
- Jenkins tests pass

**Comment by silviodonato** [comment]
+1

**Pull request merged**